core/includes/bootstrap.inc

```diff
@@ -623,9 +623,10 @@ function find_conf_path($http_host, $script_name, $require_settings = TRUE) {
     for ($i = count($uri) - 1; $i > 0; $i--) {
       for ($j = count($server); $j > 0; $j--) {
         $dir = implode('.', array_slice($server, -$j)) . implode('.', array_slice($uri, 0, $i));
-        if (isset($sites[$dir]) && file_exists(BACKDROP_ROOT . '/sites/' . $sites[$dir])) {
-          $dir = $sites[$dir];
-        }
+        if (isset($sites[$dir])) {
+          $conf = './sites/' . $sites[$dir];
+          return $conf;
+       }
```

Comment:
There is a space less in this line.

Reply:
Good catch 👍🏼 ...not sure why PHPCS didn't catch this 🤔 

Fixed with [`267dea2` (#4447)](https://github.com/backdrop/backdrop/pull/4447/commits/267dea273abb460c0705a828c728beaaf1cd3e42)